SimGeneral/MixingModule/python/mixObjects_cfi.py

```diff
@@ -119,7 +119,7 @@
     mixSimVertices.input = cms.VInputTag(cms.InputTag("famosSimHits"))
     
 mixHepMCProducts = cms.PSet(
-    makeCrossingFrame = cms.untracked.bool(False),
+    makeCrossingFrame = cms.untracked.bool(True),
     input = cms.VInputTag(cms.InputTag("generatorSmeared"),cms.InputTag("generator")),
     type = cms.string('HepMCProduct')
 )
```

Comment:
One question: are we sure that the only thing that gets written to the CrossingFrame is the protons you want to save? Or, do a whole pile of HepMCProducts end up there?

Reply:
In this implementation the MixingModule is creating a CrossingFrame on the fly with all pileup HepMC events. We then select only the particles of interest (the protons) to save to disk. With the premixing workflow this will be run only once to generate the pileup sample. The proton collection will then be copied over from the random pileup event to the main event with the data mixing module.